### PR TITLE
Create beautiful css styling

### PR DESCRIPTION
**Pull request by jason-whitted**
Create beautiful css styling

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+h1 {
+  border: 1px solid blue;
+  box-shadow: 0 0 5px red;
+  background-color: silver;
+}
```
Comment (etalisoft) [inline assets/css/style.css]: Yo dawg, could you put a newline at the end of this file?

Comment (etalisoft) [inline assets/css/style.css]: Marketing wants this to be center aligned and also 50px font.

File: index.html [hunk 2]
```diff
@@ -5,10 +5,13 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Week 7 Project</title>
+  <link rel="stylesheet" href="assets/css/style.css">
```
Comment (etalisoft) [inline index.html]: Make sure you use empty elements.

**Review by etalisoft — CHANGES_REQUESTED** [review]
Looks pretty good, but there are a few things that should get cleaned up.

File: assets/css/style.css (hunk 1 above)
Comment (jason-whitted) [reply assets/css/style.css]: Fine whatever.  H8U.

Comment (jason-whitted) [reply assets/css/style.css]: Good catch.

File: index.html (hunk 2 above)
Comment (jason-whitted) [reply index.html]: Nice!

**Review by etalisoft — APPROVED** [review]

**Pull request merged**